orca_detector/orca_params.py

```diff
@@ -65,3 +71,21 @@ class DatasetType(IntEnum):
 # Mel spectrogram hyperparameters
 FILE_MAX_SIZE_SECONDS = 10.00
 FILE_SAMPLING_SIZE_SECONDS = 0.98
+DROPOUT = 0.4
+
+# Live feed listener
+LIVE_FEED_SEGMENT_SECONDS = 1
+LIVE_FEED_SLEEP_SECONDS = 0
+LIVE_FEED_ITERATION_SECONDS = 10
+
+# TODO: Verify this is fixed to this value
+LIVE_FEED_SAMPLING_RATE = 48000
+
+# Dictionary of stream base URLs; used in building stream links
+ORCASOUND_STREAMS = {
+    'OrcasoundLab': 'https://s3-us-west-2.amazonaws.com/streaming-orcasound-net/rpi_orcasound_lab',
+    'BushPoint': 'https://s3-us-west-2.amazonaws.com/streaming-orcasound-net/rpi_bush_point',
+    'PortTownsend': 'https://s3-us-west-2.amazonaws.com/streaming-orcasound-net/rpi_port_townsend'
```

Comment:
Is this a new addition?  I didn't see it on the website last time I was there a few days ago.

Reply:
yes